styles/Board.module.css

```diff
@@ -0,0 +1,119 @@
+.container {
```

Comment:
Please, fix the spacing:

Yours:
<img width="1233" alt="Screenshot 2023-03-13 at 11 04 18 PM" src="https://user-images.githubusercontent.com/11657599/224882607-7021137f-17e2-4dde-9df2-a19dbde1c11a.png">

Prototype:
<img width="868" alt="Screenshot 2023-03-13 at 11 04 53 PM" src="https://user-images.githubusercontent.com/11657599/224882669-905683aa-1df2-4d0b-a7a9-201729329ac6.png">

Reply:
Fixed